src/pages/Main/Main.js

```diff
@@ -1,10 +1,44 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Products } from './components/Products';
+import { EventImages } from './components/EventImages';
+import './main.scss';
 
 const Main = () => {
+  const [vw, setVw] = useState(100);
+  useEffect(() => {
+    const timer = setInterval(() => {
+      if (vw <= 100 && vw > -100) {
+        setVw(prev => prev - 100);
+      }
+      if (vw <= -100) {
+        setVw(100);
+      }
+    }, 2000);
+    return () => clearInterval(timer);
+  }, [vw]);
+
   return (
-    <div>
-      <h1>main</h1>
+    <div className="main">
+      <div className="eventImages" style={{ transform: `translate(${vw}vw)` }}>
+        {EVENTIMAGES.map(({ id, src, alt }) => (
+          <EventImages key={id} src={src} alt={alt} />
+        ))}
+      </div>
```

Comment:
```suggestion
  <div className="imageWrap">
        <div
          className="eventImages"
          style={{ transform: `translateX(${currentImageIndex * -100}vw)` }}
        >
          {EVENTIMAGES.map(({ id, src, alt }) => (
            <EventImages key={id} src={src} alt={alt} />
          ))}
        </div>
      </div>
```

한가지 더 추가로 말씀드리고 싶은 부분은 위에서 리뷰 드린 방법대로 하자면
가장 처음 이미지가 가장 먼저 나와야 하는데
현재는 중간에 있는 이미지가 가장 먼저 나오기에 -100, 100 처럼 직접 계산을 해야하는 에러가 발생합니다.
이 부분은 main에 걸려 있는 justify-content: center 에 영향을 받아 발생하는 문제로 보입니다.

그래서 eventImages를 감싸는 div를 하나 주고, 거기에 hidden 속성과 width를 100vw로 해서
구현하면 첫 이미지가 가장 먼저 나오는 방식으로 구현이 가능해 집니다!

이 방법을 토대로 스타일도 수정해주세요!